app/views/home/small_bets.html.erb

```diff
@@ -0,0 +1,352 @@
+    </div>
+  </div>
+  <div class="flex items-center justify-center bg-black p-8 py-16 text-white sm:p-12 md:p-16 lg:w-1/2 xl:p-32">
+    <div class="max-w-2xl space-y-12 md:space-y-16">
+      <div class="space-y-4">
+        <h3 class="text-3xl font-medium text-purple lg:text-4xl xl:text-5xl">
+          Scale beyond your first product
+        </h3>
+        <p class="text-lg lg:text-xl xl:text-2xl">
+          Learn how successful creators built multiple income streams, from digital products to courses and memberships.
+        </p>
+      </div>
+      <div class="space-y-4">
+        <h3 class="text-3xl font-medium text-purple lg:text-4xl xl:text-5xl">
+          Get feedback
+        </h3>
+        <p class="text-lg lg:text-xl xl:text-2xl">
+          Test you next product idea with our community before investing time and energy - the "small bets" approach that works.
+        </p>
+      </div>
+      <div class="space-y-4">
+        <h3 class="text-3xl font-medium text-purple lg:text-4xl xl:text-5xl">
+          Connect with fellow creators
+        </h3>
+        <p class="text-lg lg:text-xl xl:text-2xl">
+          Join creators from music to architecture, woodworking to education - all building businesses around what they're good at.
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="w-full bg-[#f0f0f0] py-16">
+  <div class="max-w-7xl mx-auto text-center flex flex-col gap-4 px-8 pb-12">
+    <h2 class="text-5xl lg:text-6xl lg:leading-tight">Start small, Win big</h2>
+    <p class="text-xl max-w-3xl mx-auto">Perfect for creators who want to experiment with new ideas without risking everything.</p>
+  </div>
+  <div class="max-w-xl lg:max-w-7xl mx-auto px-8">
+    <div class="override grid grid-cols-1 gap-8 lg:grid-cols-3" role="list">
+      <%= render "home/shared/review_card",
+        title: "Rethought their whole side-business strategy",
+        name: "Greg Gilbert",
+        description: "I rarely recommend products publicly, but the Small Bets community offers incredible value for the price.",
+        image_url: image_url("smallbets/greggilbert.png"),
+        bg_color: "bg-pink"
+      %>
+      <%= render "home/shared/review_card",
+        title: "Went from $0 to $500 a month with KDP",
+        name: "Milly Barker",
+        description: "Watching just 6 hours of Daniel's Small Bets Fundamentals recordings has done me more good than 3+ years of striving to figure things out myself. Gratitude doesn't even come close.",
+        image_url: image_url("smallbets/millybarker.png"),
+        bg_color: "bg-yellow"
+      %>
+      <%= render "home/shared/review_card",
+        title: "Found a business partner",
+        name: "Shivam Dewan",
+        description: "Small Bets is my new idea generator. If you need inspiration and a supportive community, this is the place to be.",
+        image_url: image_url("smallbets/shivamdewan.png"),
+        bg_color: "bg-purple"
+      %>
+    </div>
```

Comment:
What do you think if make the testimonials like this instead?

https://www.figma.com/design/cc1ONs5VuP2FiwDHoTpJIT/Small-Bets-by-Gumroad-%E2%80%93-Landing-Page?node-id=260-4210&t=FWRcRhDkMaKej7C6-4

I'm finding it hard to extract a "title" out of each testimonial, but I think highlighting a phrase can be effective. We can tweak this later too, but maybe we can start with something simple like this.